xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/TemplateProviderSheet.xml

```diff
@@ -144,6 +144,46 @@
 
 {{html clean='false'}}
       &lt;/dd&gt;
+      ##
+      ## Description
+      ##
+      &lt;dt&gt;
+        &lt;label #if($isEditing)for="XWiki.TemplateProviderClass_0_description"#end&gt;$escapetool.xml($services.localization.render('xe.templateprovider.description'))&lt;/label&gt;
+      &lt;/dt&gt;
+      &lt;dd&gt;
+        #if($isEditing)
+          {{/html}}
+
+          {{html wiki='true' clean='false'}}$!providerObj.get('description'){{/html}}
```

Comment:
why use clean=false here? hmm maybe for performance reasons? If it's for performance reasons, I don't recall if we've defined an official best practice, would be great to make a proposal in this case. Thanks!